source/client/index.js

```diff
@@ -26,10 +27,13 @@ const store = createStoreWithMiddleware(reducer);
 // reduxRouterMiddleware.listenForReplays(store)
 
 ReactDOM.render(
-  <Provider store={store}>
+  <Provider store={ store }>
     <Router history={ browserHistory }>
-      <Route path="/" component={ createApp(React) } />
-      <Route path="/test-data" component={ createTestData(React) } />
+      <Route path='/' component={ createContainer(React) }>
+        <IndexRoute component={ createHome(React) } />
+        <Redirect from='/home' to='/' />
```

Comment:
Both is part of the Router API. Since the Container does not include anything but the Navbar, I put in the IndexRoute to also show the Home(App) Component on "/". So thats really handy for nested components.

The Redirect is certainly not necessary here, but its also useful fuctionality imo. For example it could later be used to redirect international URLs like "/vista-de-data" -> "/data-view" (without a real translation, though) or obsolete URLs to their new locations. 

I'd like to be able to use them later on with the easy-API as well so I put them in here in the first place. If you think thats not desireable or necessary I can leave it out for now.